cli/sub_command.go

```diff
@@ -451,7 +471,7 @@ func (c *subCmd) subscribe(p *fisk.ParseContext) error {
 			}
 			subs = append(subs, sub)
 		} else {
-			c.jsAck = false
+
```

Comment:
If we arent binding to a specific durable, we make an ephemeral consumer and there's no need for acks. Why ack it? We just show it on the stream once and thats it, hence for the `sub` behaviour being just "tail a stream" no acks are completely fine.

We only need to ack things when we bind to a consumer that has ack set, this is why its always forced off here

I guess I am just wondering why we need to support all the ack behaviours at all here

Reply:
Special case for QueuePolicy (see previous comment). For all other retention policies it does not matter indeed.